lib/commands/element.js

```diff
@@ -45,8 +45,13 @@ commands.getRect = async function (el) {
 commands.getLocation = async function (el) {
   el = el.ELEMENT ? el.ELEMENT : el;
   if (this.isWebContext()) {
-    let atomsElement = await this.useAtomsElement(el);
-    return await this.executeAtom('get_top_left_coordinates', [atomsElement]);
+    const atomsElement = await this.useAtomsElement(el);
+    const script = "return [document.body.scrollLeft, document.body.scrollTop];";
+    const [xOffset, yOffset] = await this.execute(script);
+    let loc = await this.executeAtom('get_top_left_coordinates', [atomsElement]);
```

Comment:
yes i know about this behavior, but it seems odd to mark something as `const` and then change something inside it. i consider it a weakness of `const` and is one reason i use `const` more as code documentation meaning "i don't want to change any part of this" than anything else.